embedx/identity_extension.schema.json

```diff
@@ -82,6 +82,16 @@
                   "enum": ["email"]
                 }
               }
+            },
+            "organization": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "matcher": {
```

Comment:
"match_by" would align a bit more closely with the "via" keyword, I think. 

But "matcher" could also be extended to a Jsonnet (or some other scripting) based custom matcher, that decides whether a user is part of an org or not. 

How set are you on the name?

Reply:
Organizations resolver?

Reply:
I landed on organizations matcher